### ISSUE TITLE
Error when saving config file after opening nvim-tree

### ISSUE THREAD

**Issue by Purposelessness** [LunarVim/LunarVim#3409]
### Problem description

1. Open config.lua file (via start menu or manually)
2. Open and close nvim-tree
3. Save file
4. Get the error

### LunarVim version

master-531ddb4

### Neovim version (>= 0.8.0)

NVIM v0.8.0-1210-gd367ed9b2

### Operating system/version

WSL 2 w/ Ubuntu 22.04.1 LTS

### support info

```console
# LspInfo
 Language client log: /home/purposelessness/.local/state/nvim/lsp.log
 Detected filetype:   lua
 
 1 client(s) attached to this buffer: 
 
 Client: sumneko_lua (id: 1, pid: nil, bufnr: [3])
 	filetypes:       lua
 	autostart:       true
 	root directory:  Running in single file mode.
 	cmd:             lua-language-server
 
 Configured servers list: sumneko_lua
```
```console
# LvimInfo
Buffer info
* filetype:                lua
* bufnr:                   3
* treesitter status:       active

Active client(s)
* name:                      sumneko_lua
* id:                        1
* supported filetype(s):     lua
* attached buffers:          3
* root_dir pattern:          3
* capabilities:              documentHighlightProvider  | colorProvider                   | foldingRangeProvider
                             documentFormattingProvider | workspaceSymbolProvider         | hoverProvider
                             documentSymbolProvider     | typeDefinitionProvider          | referencesProvider
                             definitionProvider         | documentRangeFormattingProvider

Automatic LSP info
* Skipped servers: [angularls, ansiblels, csharp_ls, cssmodules_ls, denols, ember, emmet_ls, eslint, glint, golangci_lint_ls, gradle_ls, graphql, jedi_language_server, ltex, phpactor, psalm, pylsp, quick_lint_js, reason_ls, rome, ruby_ls, solang, solc, sorbet, sourcery, sqlls, sqls, stylelint_lsp, svlangserver, tflint, verible, vuels]

Formatters info
* Active: 
* Supported: [lua_format, stylua]

Linters info
* Active: 
* Supported: [luacheck, selene]

Code actions info
* Active: 
```

### Screenshots

![image](https://user-images.githubusercontent.com/64734689/199837549-1452b62d-fac3-4b8c-9b0d-fbef9f139a97.png)

**Comment by Purposelessness** [comment]
I managed to fix the error by changing the line in `core/theme.lua` to this:
![image](https://user-images.githubusercontent.com/64734689/199838865-b2630e38-bf69-45a5-bf86-b8594531f28f.png)

**Comment by ChristianChiarulli** [comment]
@kylo252 did we fix this?

**Comment by kylo252** [comment]
fixed in #3416
